edx_api/integration_test.py

```diff
@@ -395,3 +395,21 @@ def test_user_info_timeout():
         get.side_effect = mocked_timeout
         with pytest.raises(Timeout):
             user_info.get_user_info()
+
+
+@require_integration_settings
+def test_enrollments_list():
+    """
+    Enrolls the user in a course and then pulls down the enrollments for the user.
+    This assumes that the course in the edX instance is available for enrollment.
+    """
+    api = EdxApi({'access_token': ACCESS_TOKEN}, base_url=BASE_URL)
+    enrollments = api.enrollments.get_enrollments()
+    print(enrollments)
+
+    enrolled_courses = [
+        enrolled_course.course_details.course_id
+        for enrolled_course in enrollments.enrolled_courses
+    ]
+
+    assert 'course-v1:edX+DemoX+Demo_Course' in enrolled_courses
```

Comment:
Could we check that all the enrollments are returned?
Since this would still succeed if only the first one is returned, it might not be the most comprehensive test.

Reply:
got it.

Reply:
Looks like these tests haven't been updated yet.
It'd also be nice to see some testing of the pagination in here.